app/inputs/transaction_chosen_input.rb

```diff
@@ -21,7 +26,7 @@ def label
   private
 
   def collection_items
-    template.instance_variable_get("@#{attribute_name}")
+    template.instance_variable_get("@#{attribute_name}").to_a
```

Comment:
I was going back and forth by adding the `to_a` here, and then using `collection_items.size` throughout the `input` method instead of the `collection_size` variable. I eventually settled on leaving it as a variable, but guess I forgot to remove this one.